<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>703</width>
    <height>578</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>330</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget {
	background: qlineargradient(
    	x1: 0, y1: 1, x2: 0, y2: 0,
    	stop: 0.0 hsl(39, 76%, 75%),
    	stop: 0.31 hsl(39, 28%, 55%),
    	stop: 1.0 hsl(216, 18%, 16%)
  );
	border-radius: 10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="topMargin">
     <number>60</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="logo_img">
      <property name="maximumSize">
       <size>
        <width>161</width>
        <height>161</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/icons.qrc">:/icons/icons/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="logo_label">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #222831;</string>
      </property>
      <property name="text">
       <string>FitWrap</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>320</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_2 {
	background-color: #EEEEEE;
	border-radius: 10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>36</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QFrame" name="top_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="minimize_window_button">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat Medium</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#minimize_window_button {
	background-color: #00CA4E;
	border-radius: 7px;
	text-align: center;
	margin-bottom: 1px;
}

QPushButton#minimize_window_button:hover {
	background-color: #00ff62;
}

QPushButton#minimize_window_button:pressed {
	background-color: #1EB859;
}</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_window_button">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat Medium</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#close_window_button {
	background-color: #FF605C;
	border-radius: 7px;
	text-align: center;
	margin-bottom: 1px;
}

QPushButton#close_window_button:hover {
	background-color: #FF8C89;
}

QPushButton#close_window_button:pressed {
	background-color: #B7221F;
}</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="page">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="log_in_page">
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>275</width>
          <height>310</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 2px solid rgba(0, 0, 0, 0);
	border-bottom-color: #2F4858;
	background-color: rgba(0, 0, 0, 0);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="log_in_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>55</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>252</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="login_form">
          <item>
           <widget class="QLineEdit" name="login_field">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_field">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="remember_me_ckbox">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Remember me</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="login_error_label">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat Medium</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: red;</string>
            </property>
            <property name="text">
             <string>  The login or password is incorrect.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sign_in_btn">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#sign_in_btn {
	background-color: #F0CF90;
	color: black;
	border-radius: 5px;
}

QPushButton#sign_in_btn:hover {
	background-color: #F3D8A5;
}

QPushButton#sign_in_btn:pressed {
	background-color: #C3943C;
}</string>
            </property>
            <property name="text">
             <string>Sign in</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="create_acc_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>101</width>
           <height>24</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Create an account</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="register_page">
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>295</width>
          <height>326</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 2px solid rgba(0, 0, 0, 0);
	border-bottom-color: #2F4858;
	background-color: rgba(0, 0, 0, 0);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="register_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>55</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>252</width>
           <height>244</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="register_form">
          <item>
           <widget class="QLineEdit" name="login_register_field">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="login_warning_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>15</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat Medium</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: red;</string>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="text">
             <string>  Login warning placeholder.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_1_field">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password_2_field">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Confirm password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passwd_warning_label">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Montserrat Medium</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: red;</string>
            </property>
            <property name="text">
             <string>  Password warning placeholder</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="register_btn">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>40</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#register_btn {
	background-color: #F0CF90;
	color: black;
	border-radius: 5px;
}

QPushButton#register_btn:hover {
	background-color: #F3D8A5;
}

QPushButton#register_btn:pressed {
	background-color: #C3943C;
}</string>
            </property>
            <property name="text">
             <string>Register</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>login_warning_label</zorder>
         <zorder>login_register_field</zorder>
         <zorder>password_1_field</zorder>
         <zorder>password_2_field</zorder>
         <zorder>register_btn</zorder>
         <zorder>passwd_warning_label</zorder>
        </widget>
        <widget class="QPushButton" name="alrdy_have_acc_btn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>131</width>
           <height>24</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Already have an account</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>widget_2</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
